### PR TITLE
Hello-rails-react

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, I

- [ ] created a new Rails app called hello-rails-react with PostgreSQL database,
- [ ] installed needed node dependencies,
- [ ] created a babel.config.js in the root folder,
- [ ] updated webpack.config.js to include the babel-loader,
- [ ] created a table for storing messages and created 5 different greetings.
- [ ] created an API endpoint that selects a random greeting from the greetings table,
- [ ] created a static view that will be the root of the app.
- [ ] created the App component with react-router and rendered it in the static view.
- [ ] created the Greeting component that displays a greeting,
- [ ] settled up a route for the Greeting component,
- [ ] created a store, an action, and a reducer that connects to the API endpoint to get the random greeting, and
- [ ] displayed the random greeting in the Greeting component.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  gem 'web-console'
+
+  # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
+  # gem "rack-mini-profiler"
+
+  # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
+  # gem "spring"
+end
+
+group :test do
```
Comment (rloterh) [inline Gemfile:71]: - [ ] Unable to load API data to the React frontend due to CORS not properly configured. Kindly ensure to add rack-cors gem as well as add the appropriate CORS configurations.

CORS Error:
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Yodit, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🚦 _ 🏡 🏃‍♂️ 🏚️ 🌴 🗼

## Highlights 👍  
- Good job so far with implementing backend and React frontend in on App!
- ReadMe is professional!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Gemfile (hunk 1 above)
Comment (yodit93) [reply Gemfile:71]: Ok, I will 
Thank you

**Review by gemmen29 — APPROVED** [review]
Hi @yodit93, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yodit93** [comment]
Thank you 🙌

**Pull request merged**